configurator.go

```diff
@@ -97,6 +97,13 @@ type ProxyConfig struct {
 	Port		string	`json:"port"`
 }
 
+type CDNHostInfo struct {
```

Comment:
Do we need to add this in the main 'Configurations' ?

Workflow is we parse the Configurations first and then look at individual ones.